src/pages/SwapList/SwapStatusIcon.tsx

```diff
@@ -32,38 +32,46 @@ function SwapStatusIcon({ status }: SwapStatusIconProps) {
   switch (status) {
     case Status.InProgress:
       return (
-        <Tooltip title="Swap in progress" placement="bottom-start">
-          <FontAwesomeIcon
-            icon={faSync}
-            className={classes.inProgressIcon}
-            size="lg"
-          />
+        <Tooltip title="Swap in progress">
+          <div>
```

Comment:
❓ Are these `div`s relevant for the change? Just wondering :)

Reply:
Not to the "Clearer sell/buy" side of things, no. It was a minor [commit](https://github.com/comit-network/comit-i/pull/103/commits/b5c68f370d394c29d5cf3495a059d50ef650d558) I added because I noticed that something was broken.